Formula/libfyaml.rb

```diff
@@ -0,0 +1,50 @@
+class Libfyaml < Formula
+  desc "Fully feature complete YAML parser and emitter"
+  homepage "https://github.com/pantoniou/libfyaml"
+  url "https://github.com/pantoniou/libfyaml/archive/refs/tags/v0.7.12.tar.gz"
+  sha256 "864e60dc1863e2ba2048b9765d9addd653f6c387e835438f09cb5f26a2a352e1"
```

Comment:
The official source tarball is probably better as it includes the generated configure script, which is our standard way of building stable autotools-based packages:
```suggestion
  url "https://github.com/pantoniou/libfyaml/releases/download/v0.7.12/libfyaml-0.7.12.tar.gz"
  sha256 "485342c6920e9fdc2addfe75e5c3e0381793f18b339ab7393c1b6edf78bf8ca8"
```